tools/BackPopulateSNRAndAcquisitionOrder.pl

```diff
@@ -89,7 +99,7 @@
 ################## Instantiate MRIProcessingUtility ############
 ################################################################
 my $utility = NeuroDB::MRIProcessingUtility->new(
-                  \$dbh,$debug,$TmpDir,$logfile,
+                  $db, \$dbh,$debug,$TmpDir,$logfile,
```

Comment:
so here you are adding`$db` and you keep `$dbh` because not all functions are refactored yet, but when we do, we will keep one of the two.
A comment for us to remember this later on!